### PR TITLE
Add alt text for lab member headshots

### PR DESCRIPTION
**Pull request by LDubya**
This commit adds alt text to lab member headshots. The alt texts were contributed by many lab members.

### CODE REVIEW

File: _members/mimi-alkattan.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Mimi Alkattan
 name_degree: Mimi Alkattan, MS
 photo: mimi-alkattan.jpg
+alt: Photo of a smiling young woman who appears to be Asian with dark hair and brown eyes wearing a white shirt and a peach-colored scarf. She is standing in front of an ornate building and garden with a cloudy sky.
```
Comment (lchoy) [inline _members/mimi-alkattan.md:5]: alt: Photo of a smiling young woman with dark hair and brown eyes wearing a white shirt and a peach-colored scarf. She is standing in front of an ornate building and garden with a cloudy sky.

I could be mistaken, but I don't recall Mimi having Asian roots.

File: _members/zoey-ho.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: Zoey Ho
 name_degree: Zoey Ho, BS, MBI
 
 photo: zoey-ho.jpg
+alt: Headshot of a young woman who appears to be Asian with medium-length black hair and an open smile wearing a bright yellow top exposing her shoulders. The woman is outside in sunny weather in front of a lawn with a blurry brick building in the background.
```
Comment (lchoy) [inline _members/zoey-ho.md:6]: alt: Headshot of a young woman who appears to be Asian, with medium-length black hair and an open smile, wearing a bright yellow top exposing her shoulders. The woman is outside in sunny weather, in front of a lawn with a blurry brick building in the background.

File: _members/yousry-elsadec.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: Mohamed Yousry ElSadec
 name_degree: Mohamed Yousry ElSadec
 
 photo: yousry-elsadec.jpg 
+alt: Photo of a young male who appears to be Asian with shortly cut dark curly hair and a beard. He’s wearing a purple button-up and smiling from a window into the distance. In the background we can see the location is by a body of water.
```
Comment (lchoy) [inline _members/yousry-elsadec.md:6]: Add comma after the word "background".

File: _members/undina-gisladottir.md [hunk 4]
```diff
@@ -3,6 +3,7 @@ title: Undina Gisladottir
 name_degree: Undina Gisladottir, BS, MBI
 
 photo: undina-gisladottir.jpg
+alt: Photo of a young female who appears to be white smiling with long straight blonde hair. She is wearing a high-neck short sleeve shirt, in front of a light color wooden fence.
```
Comment (lchoy) [inline _members/undina-gisladottir.md:6]: Should there be a comma before the word "smiling"?

File: _members/theresa-harbig.md [hunk 5]
```diff
@@ -3,6 +3,7 @@ title: Theresa Anisja Harbig
 name_degree: Theresa Anisja Harbig, MSc
 
 photo: Theresa_Harbig.jpeg
+alt: Headshot of a smiling young woman who appears to be white in front of a wooden panel. She has shoulder length brown, slightly wavy hair and is wearing large round brown glasses and small circular earrings. A large blue knitted scarf is around her neck and she’s indoors wearing a sweater in a slightly different shade of blue/green
```
Comment (lchoy) [inline _members/theresa-harbig.md:6]: Missing period on last sentence.

File: _members/jennifer-marx.md [hunk 6]
```diff
@@ -3,6 +3,7 @@ title: Jennifer K Marx
 name_degree: Jennifer K Marx, MEng
 
 photo: jkmarx.png
+alt: Headshot of a young, light-skinned woman taken in front of a brick wall. She has dark, medium-long hair, dark eyes and has a bright-white smile. She’s wearing a green t-shirt and has a small silver ring on a necklace around her neck
```
Comment (lchoy) [inline _members/jennifer-marx.md:6]: Missing period on last sentence.

File: _members/carolina-nobre.md [hunk 7]
```diff
@@ -3,6 +3,7 @@ title: Carolina Nobre
 name_degree: Carolina Nobre, MSc
 
 photo: carolina-nobre.jpg
+alt: Selfie of a young woman who appears to be white in front of a window. She has long, straight, black hair, dark eyes and is slightly smiling. She has a small golden nose piercing and is wearing a teal shirt
```
Comment (lchoy) [inline _members/carolina-nobre.md:6]: Missing period on last sentence.

**Review by lchoy — COMMENTED** [review]
Hi Larry, 

Thanks for adding all the alt text! I had a few minor suggestions. I noticed a few sentences were missing periods at the end. I also suggested some commas to assist with readability.

File: _members/mimi-alkattan.md (hunk 1 above)
Comment (LDubya) [reply _members/mimi-alkattan.md]: I grouped by continent for consistency, so individuals with a likely Middle Eastern heritage were included under the Asian description. However, your description works too. Thank you for the suggestion!

**Comment by LDubya** [comment]
New changes are pushed, and all comments are resolved.

**Review by john-conroy — APPROVED** [review]
Stepping into approve while Lisa is out. @LDubya with this approval you should be able to merge when ready.

**Pull request merged**